tests/test_phase_psf.py

```diff
@@ -541,72 +569,134 @@ def test_phase_gradient_shoot():
 
     if __name__ == '__main__':
         exptime = 15.0
+        time_step = 0.05
         centroid_tolerance = 0.05
-        second_moment_tolerance = 0.5
+        size_tolerance = 0.05  # absolute
+        size_bias = 0.02  # as a fraction
+        shape_tolerance = 0.005
     else:
-        exptime = 0.2
-        centroid_tolerance = 0.2
-        second_moment_tolerance = 1.5
+        exptime = 1.0
+        time_step = 0.1
+        centroid_tolerance = 0.3
+        size_tolerance = 0.3
+        size_bias = 0.15
+        shape_tolerance = 0.04
 
     psfs = [atm.makePSF(lam, diam=diam, theta=th, exptime=exptime, aper=aper) for th in thetas]
+    psfs2 = [atm2.makePSF(lam, diam=diam, theta=th, exptime=exptime, aper=aper, time_step=time_step)
+             for th in thetas]
```

Comment:
For the photon shooting, I use the exact time for every photon instead of a discretized time like is required for Fourier drawing.  (It's easy to implement, and probably the right thing to do, though I doubt there is any appreciable difference compared to using discretized times as long as the time interval is small).

I agree the atms should be exactly equal at this point.

If there is boiling, then photon-shooting is careful to update the screens periodically, but the flow is still frozen in between these updates, and exact times are still used.  Note that the time_step used for updating the screens in this case is the one passed to the Atmosphere or AtmosphericScreen though, and not the one passed to `makePSF` or `PhaseScreenPSF`, which are never used for photon shooting.